terraform/groups/ecs-service/main.tf

```diff
@@ -33,6 +33,17 @@ module "ecs-service" {
   lb_listener_arn                 = data.aws_lb_listener.service_lb_listener.arn
   lb_listener_rule_priority       = local.lb_listener_rule_priority
   lb_listener_paths               = local.lb_listener_paths
+  multilb_setup                   = true
+  multilb_listeners               = {
```

Comment:
If you make the changes I've suggested above, you should be able to use this, but please adjust the indentation:

```
multilb_listeners  = {
    "pub-lb-http": {
      listener_arn             = data.aws_lb_listener.service_lb_listener_80.arn,
      load_balancer_arn  = data.aws_lb.service_lb.arn
    }
    "pub-lb-https": {
      listener_arn             = data.aws_lb_listener.service_lb_listener_443.arn
      load_balancer_arn  = data.aws_lb.service_lb.arn
    }
  }
```